### DISCUSSION TITLE
Verifying one domain with two Vercel accounts

### DISCUSSION THREAD

**Discussion by grumpyp** [vercel/community#4831] [Help] [answered]
### Summary

Hi,

I have two different accounts and deployments done with Vercel.

Now I have a domain which uses the main domain for one deployment and I wanna use a subdomain for the other accounts deployment.

When I put the TXT entry to my DNS settings I run into the conflict.

Is there another way to do this?

### Steps to Reproduce

Summary <-

**Comment by amyegan** [comment]
Sorry you're having trouble with your domain. Can you tell me more about the conflict you're experiencing? Screenshots or a quick video may be helpful to show what's going wrong.

**Comment by grumpyp** [comment]
Hi,

basically I am adding two TXT records - but then Google-Domains shows an error basically.

The "error" asks me if I want to replace the current entry

<img width="998" alt="image" src="https://github.com/vercel/community/assets/75830792/b79c2947-9e4d-4f5d-b083-afa889eac9e1">

<img width="616" alt="image" src="https://github.com/vercel/community/assets/75830792/8a2374da-037d-46ae-bead-5f4c9984a60a">

**Reply by TooTallNate** [reply, marked as answer]
You can delete the older `_vercel` TXT file entry. It's only needed for the verification process when the subdomain is being assigned to the 2nd Vercel account. After that verification is complete, it is safe to delete that TXT entry.
